doc/install.rst

```diff
@@ -62,8 +64,15 @@ It is also possible to install pyOptSparse by calling ``python setup.py install`
 .. note::
   Some optimizers are proprietary and their sources are not distributed with pyOptSparse.
   To use them, please follow the instructions on specific optimizer pages.
-  
-For those who intend to use pyOptSparse with IPOPT, OpenMDAO developers provide a `bash script <https://github.com/OpenMDAO/build_pyoptsparse>`_ that simplifies the installation of the optimizer with different linear solvers.
+
+For those who intend to use pyOptSparse with IPOPT, OpenMDAO developers provide a `bash script <https://github.com/OpenMDAO/build_pyoptsparse>`_ that simplifies the installation of the optimizer with different linear solvers. With IPOPT installed, set the environment variable ``IPOPT_DIR`` to the path of the IPOPT installation. The installation directory structure is expected to look like the following:
```

Comment:
The IPOPT instructions should be moved to the IPOPT page, and merged with the content there.

Reply:
Sounds good. Will get to it sometime next week I think